ui-v2/app/services/repository/type/event-source.js

```diff
@@ -17,10 +17,13 @@ const createProxy = function(repo, find, settings, cache, serialize = JSON.strin
     const meta = get(event.data || {}, 'meta') || {};
     if (typeof meta.date !== 'undefined') {
       // unload anything older than our current sync date/time
-      store.peekAll(repo.getModelName()).forEach(function(item) {
-        const date = get(item, 'SyncTime');
-        if (typeof date !== 'undefined' && date != meta.date) {
-          store.unloadRecord(item);
+      store.peekAll(repo.getModelName()).forEach(function(item, i) {
```

Comment:
Looks like `i` is unused.

Reply:
😄  ah yes!

In all likelihood this PR will never go down and be closed. It was incase we end up releasing 1.7.0 final without namespace support.

When we realized the UI namespace wasn't going to make it into the beta, we PRed this to cover this bug which is fixed in the namespace branch (and also takes namespaces into account). We were unsure whether this bug was important enough to include in the beta (hence the PR) and the team deemed it not to be, but I'm leaving this PR hanging here just incase. Once https://github.com/hashicorp/consul/pull/6639 is merged to `ui-staging` and then `master` I can close this.